tools/actions/commit-queue.sh

```diff
@@ -106,13 +83,14 @@ for pr in "$@"; do
       continue
     fi
   else
-    # If there's only one commit, we can use the Squash and Merge feature from GitHub
+    # If there's only one commit, we can use the Squash and Merge feature from GitHub.
+    # TODO: use `gh pr merge` when the GitHub CLI allows to customize the commit title (https://github.com/cli/cli/issues/1023).
     jq -n \
       --arg title "$(git log -1 --pretty='format:%s')" \
       --arg body "$(git log -1 --pretty='format:%b')" \
       '{merge_method:"squash",commit_title:$title,commit_message:$body}' > output.json
     cat output.json
-    gitHubCurl "$(mergeUrl "$pr")" PUT --data @output.json > output
+    gh api -X PUT "$(mergeUrl "$pr")" --input output.json > output
```

Comment:
Let's add an extra check to avoid silently crashing the CQ if the merge fails.
```suggestion
    if ! gh api -X PUT "$(mergeUrl "$pr")" --input output.json > output; then
      commit_queue_failed "$pr"
      continue
    fi
```

Reply:
```suggestion
    GITHUB_TOKEN=$GITHUB_TOKEN gh api -X PUT "$(mergeUrl "$pr")" --input output.json > output
```

Reply:
A combination of both suggestions:
```suggestion
    if ! GITHUB_TOKEN=$GITHUB_TOKEN  gh api -X PUT "$(mergeUrl "$pr")" --input output.json > output; then
      commit_queue_failed "$pr"
      continue
    fi
```